internal/cli/events/organization/list.go

```diff
@@ -87,10 +84,7 @@ func ListBuilder() *cobra.Command {
 		Aliases: []string{"ls"},
 		Args:    require.NoArgs,
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			if opts.orgID != "" && opts.projectID != "" {
-				return fmt.Errorf("both --%s and --%s set", flag.ProjectID, flag.OrgID)
-			}
-			if opts.orgID == "" && opts.projectID == "" {
+			if opts.orgID == "" {
```

Comment:
can't highlight all the lines but all of these can now be 
```suggestion
return opts.PreRunE(
				opts.ValidateOrgID,
				opts.initStore,
				opts.InitOutput(cmd.OutOrStdout(), createTemplate),
			)
```